community-build/test/scala/dotty/communitybuild/CommunityBuildTest.scala

```diff
@@ -26,7 +53,7 @@ class CommunityBuildTest {
    *  @param updateCommand  The sbt command used to update the project
    *  @param extraSbtArgs   Extra arguments to pass to sbt
    */
-  def test(project: String, testCommand: String, updateCommand: String, extraSbtArgs: Seq[String] = Nil): Unit = {
+  def test(project: String)(commandAndArgs: => (String, Seq[String])): Unit = {
```

Comment:
Why is commandAndArgs by-name?

Reply:
No particular reason – just following a pattern of implementing control structures (`test`) via curried functions. If it works with by-value, do you think this should be switched to by-value?

Reply:
If there is nothing relying on it I would make it by-value